.github/workflows/crowdin-sync.yml

```diff
@@ -0,0 +1,29 @@
+name: Crowdin Synchronisation
+
+permissions:
+  contents: write
+  pull-requests: write
+
+on:
+  push:
+    branches: [main]
```

Comment:
`next` is our default branch, should we add `next` here too?

Reply:
> `next` is our default branch, should we add `next` here too?

@siljec Updated to include `next` now too 👍